packages/backend-modules/auth/lib/Sessions.js

```diff
@@ -12,12 +13,20 @@ const InitiateSessionError = newAuthError(
 )
 const NoSessionError = newAuthError('no-session', 'api/token/invalid')
 
-const destroySession = async (req) => {
+const destroySession = async (req, res) => {
   return new Promise((resolve, reject) => {
     req.session.destroy((error) => {
       if (error) {
         return reject(new DestroySessionError({ headers: req.headers, error }))
       }
+      if (res) {
+        const cookieOptions = getCookieOptions()
+        res.clearCookie(process.env.COOKIE_NAME ?? 'connect.sid', cookieOptions) // clear session cookie
+        res.clearCookie(
+          process.env.JWT_COOKIE_NAME ?? 'republik-token',
+          cookieOptions,
+        ) // clear jwt cookie
```

Comment:
```suggestion
        res.clearCookie(process.env.COOKIE_NAME ?? 'connect.sid', cookieOptions)
        res.clearCookie(
          process.env.JWT_COOKIE_NAME ?? 'republik-token',
          cookieOptions,
        )
```
what information do those code comments add?